### ISSUE TITLE
Docker Hub shows the GitHub repository's README instead of the description in Docker Cloud

### ISSUE THREAD

**Issue by philippgille** [philippgille/hello-netcoreapp#21]
Description in Docker Cloud:

```
The build is automated on Docker Cloud, which pushes the images to the Docker Hub registry. This is because Docker Hub doesn't support multi-stage Dockerfile builds as of now (2017-07-08), which is also the reason why the automated build appears to fail. The images pushed by Docker Cloud are automated and work fine though.

To learn about hello-netcoreapp, visit [https://github.com/philippgille/hello-netcoreapp](https://github.com/philippgille/hello-netcoreapp)
```

**Comment by philippgille** [comment]
According to the documentation, the README in the same directory of the Dockerfile gets used, see [https://docs.docker.com/docker-hub/builds/#understand-the-build-process](https://docs.docker.com/docker-hub/builds/#understand-the-build-process).

But for example in [aspnet/aspnet-docker](https://github.com/aspnet/aspnet-docker) there are 3 README files in the root of the repository and it seems as if Docker Hub shows the one containing the name of the Docker Hub repository. `README.aspnetcore.md` gets shown in [https://hub.docker.com/r/microsoft/aspnetcore/](https://hub.docker.com/r/microsoft/aspnetcore/) and `README.aspnetcore-build.md` gets shown in [https://hub.docker.com/r/microsoft/aspnetcore-build/](https://hub.docker.com/r/microsoft/aspnetcore-build/).

**Comment by philippgille** [comment]
Can be solved with #5
